### PR TITLE
Updated snackbar and routing in App.vue privacy-po

### PR DESCRIPTION
**Pull request by freedannyv**
#394 

[ ] Bug Fix
[ ] Refactor
[ ] New Feature
[ ] Update to Existing Feature
[x ] Other (state below)

# Description
Updated the snackbar as there was a duplicate key. Props remains titled showSnackBar and methods and data referencing the snackbar are renamed from showSnackBar to isSnackBar.

App.vue updated URL to redirect to privacy-policy.
# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Log in as '...'
2. Go to 'www.foodouken.com' and test Agree link to direct to privacy-policy. Check no duplicated keys errors occur when opening foodouken (duplicated keys error fixed)
3. Click on '...'
4. See the updated element x

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -27,7 +27,7 @@
           <a
             class="underline cursor-pointer"
             target="_blank"
-            href="https://hub.paulchrisluke.com/-temporary-slug-7a760197-2d5d-4314-876c-ade5923d6dd8"
+            href="/privacy-policy"
```
Comment (giteshk11) [inline src/App.vue:30]: @freedannyv You have to redirect using the router.
For your reference
https://github.com/whynotearth/foodouken.com/blob/503619575af6f2b1f8d3d4462318b089cf388d4f/src/layouts/Default.vue#L16-L19

**Review by paulchrisluke — APPROVED** [review]

File: src/components/SnackBar.vue [hunk 2]
```diff
@@ -17,10 +17,28 @@ export default {
...
+      this.isSnackBar = false;
+    } else {
+      this.isSnackBar = true;
+    }
+  },
+  methods: {
+    setsnackBarCookie: function() {
+      this.isSnackBar = false;
+      localStorage.setItem('hasConsent', true);
+    }
```
Comment (giteshk11) [inline src/components/SnackBar.vue:38]: This code is tightly coupled now with the privacy-policy snack bar, so now it cannot be used somewhere else.
Please move this code out of this component and use the props to hide or show the snack bar.
Let me know if you have any issues, I'll explain it again.

**Review by giteshk11 — CHANGES_REQUESTED** [review]
Request some changes @freedannyv

File: src/components/SnackBar.vue (hunk 2 above)
Comment (freedannyv) [reply src/components/SnackBar.vue:38]: I'm sorry, I have a general idea but I'm really not sure how to implement it.

**Pull request closed without merging**